src/main/java/com/ldtteam/structurize/Structurize.java

```diff
@@ -42,9 +44,23 @@ public Structurize()
         Mod.EventBusSubscriber.Bus.FORGE.bus().get().register(EventSubscriber.class);
         DistExecutor.unsafeRunWhenOn(Dist.CLIENT, () -> () -> Mod.EventBusSubscriber.Bus.FORGE.bus().get().register(ClientEventSubscriber.class));
 
-        if (DataVersion.CURRENT == DataVersion.UPCOMING)
+        if (DataFixesManager.getDataFixer() instanceof com.mojang.datafixers.DataFixerUpper)
```

Comment:
Can you move that into DataFixerUtils as static boolean and display text in buildtool (like the escape one)?